packages/amplication-client/src/Components/FeatureIndicator.tsx

```diff
@@ -0,0 +1,105 @@
+import { EnumTextColor, Icon } from "@amplication/ui/design-system";
+import Tooltip, { TooltipProps, tooltipClasses } from "@mui/material/Tooltip";
+import { styled } from "@mui/material/styles";
+import { useCallback, useContext } from "react";
+import { Link, useHistory } from "react-router-dom";
+import { useTracking } from "react-tracking";
+import { AppContext } from "../context/appContext";
+import { AnalyticsEventNames } from "../util/analytics-events.types";
+import "./FeatureIndicator.scss";
+import { IconType } from "./FeatureControlContainer";
+
+const WarningTooltip = styled(
+  ({ className, placement, ...props }: TooltipProps) => (
+    <Tooltip {...props} placement={placement} classes={{ popper: className }} />
+  )
+)(({ theme }) => ({
+  [`& .${tooltipClasses.arrow}`]: {
+    color: "#53dbee",
+  },
+  [`& .${tooltipClasses.tooltip}`]: {
+    backgroundColor: "#15192C",
+    color: "#ffffff",
+    maxWidth: 360,
+    border: "1px solid #53dbee",
+    borderRadius: "4px",
+    padding: "6px 8px",
+    fontFamily: "unset",
+    fontSize: "10px",
+    fontWeight: "unset",
+  },
+}));
+
+const CLASS_NAME = "amp-locked-feature-indicator";
+
+type Props = {
+  featureName: string;
+  icon?: IconType;
+  comingSoon?: boolean;
+  placement?: TooltipProps["placement"];
+  tooltipIcon?: string;
+  text?: string;
+  linkText?: string;
+};
+
+export const FeatureIndicator = ({
+  featureName,
+  icon = IconType.Lock,
+  comingSoon = false,
+  tooltipIcon,
+  placement = "top-start",
+  text = "Available as part of the Enterprise plan only.",
+  linkText = "Upgrade",
+}: Props) => {
+  const history = useHistory();
+  const { trackEvent } = useTracking();
+  const { currentWorkspace } = useContext(AppContext);
+
+  const handleViewPlansClick = useCallback(() => {
+    history.push(`/${currentWorkspace.id}/purchase`, {
+      from: { pathname: window.location.pathname },
+    });
+    trackEvent({
+      eventName: AnalyticsEventNames.ProFeatureLockClick,
+      featureName,
+    });
+  }, [currentWorkspace, window.location.pathname]);
+
+  return (
+    <WarningTooltip
+      placement={placement}
+      title={
+        <div className={`${CLASS_NAME}__tooltip__window`}>
```

Comment:
please take this component out and pass the props you need

Reply:
It's not a new component. I just changed its name. I don't know why we did it like that, but it's not a part of my pr. It was named before: LockFeatureIndicator, but now we need it not only when the feature is disabled so I removed the "locked" part